apiserver/facades/agent/uniter/newlxdprofile.go

```diff
@@ -307,16 +309,16 @@ func (u *LXDProfileAPIv2) getOneCanApplyLXDProfile(machine LXDProfileMachineV2,
 	return false, nil
 }
 
-func (u *LXDProfileAPIv2) getModelTypeProviderType(ctx context.Context) (string, state.ModelType, error) {
-	m, err := u.backend.Model()
+func (u *LXDProfileAPIv2) getModelTypeProviderType(ctx context.Context) (string, model.ModelType, error) {
```

Comment:
I think this func can safely die in a blaze of glory. One call to the model info service gets you both pieces of information. That is the model type and the cloud type. You don't need to read model config for this info.

Reply:
perfect, then I don't actually need model config at all for the LXD profile API